smart-contracts/scripts/versioned-solidity-files.sh

```diff
@@ -0,0 +1,91 @@
+#! /bin/sh
+
+# all the commits  
```

Comment:
Please add a comment at the top of the file to indicate that this is a script to flatten all contract codes so we can use them for the upgrades?

Reply:
Finally I moved the script to `past-versions` directly so it is less confusing. Also renamed to `flatten-past-versions.sh` and added a comment to the file